### DISCUSSION TITLE
Capacitor

### DISCUSSION THREAD

**Discussion by spofastic** [dokuhn/openDTU-BreakoutBoard#10] [Q&A] [answered]
Hi Dominik,

nice piece of work. Could you specify the capacitator? Is it 10uF ?v?

Thx, Marc

**Comment by dokuhn** [comment, marked as answer]
Thank you Marc for your feedback about the specification of the capacitor. Since several people asked for it, as well as for the pinout of the used MCU board, I described both in the [README.md ](https://github.com/dokuhn/openDTU-BreakoutBoard/) file of the project now. I hope this helps you a little bit. 

with best regards, Dominik
